projects/kubernetes/kubernetes/build/create_binaries.sh

```diff
@@ -26,9 +26,9 @@ source "${MAKE_ROOT}/build/lib/init.sh"
 source "${MAKE_ROOT}/../../../build/lib/common.sh"
 
 GIT_TAG="$(cat ${MAKE_ROOT}/${RELEASE_BRANCH}/GIT_TAG)"
+RELEASE_FILE="${MAKE_ROOT}/${RELEASE_BRANCH}/RELEASE"
 PATCH_DIR=${MAKE_ROOT}/${RELEASE_BRANCH}/patches
-export KUBE_GIT_VERSION_FILE="${MAKE_ROOT}/${RELEASE_BRANCH}/KUBE_GIT_VERSION_FILE"
-
+export KUBE_GIT_VERSION=$(build::version::kube_git_version $GIT_TAG $RELEASE_FILE $RELEASE_BRANCH)
```

Comment:
The only key that should matter during the build time process is the git_version since its reflected in the final binary.  The make the process of creating a new release easier, avoid requiring this file to exist and let the build process create it later.